src/test/java/elizCurtnazar/tests/tema6/HerokuForms.java

```diff
@@ -0,0 +1,138 @@
+package elizCurtnazar.tests.tema6;
+
+import elizCurtnazar.driver.BrowserManager;
+import org.apache.commons.io.FileUtils;
+import org.openqa.selenium.By;
+import org.openqa.selenium.OutputType;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.remote.Browser;
+import org.openqa.selenium.support.ui.Select;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.List;
+
+public class HerokuForms {
+
+    static ChromeDriver driver = null;
+
+    public static void main(String[] args) throws IOException {
+        navigateToHerokuFormsPage();
+        inputUsername();
+        inputPassword();
+        inputTextAreaComment();
+        chooseFile();
+        checkboxItems();
+//        radioItems();
+        try {
+            radioItems();
+            System.out.println("Am intrat pe blocul de try.");
+        } catch (IndexOutOfBoundsException e) {
+            System.out.println("Am intrat de blocul de catch");
+            List <WebElement> radioButtons = driver.findElements(By.name("radioval"));
+            radioButtons.get(2).click();
+            System.out.println("Am ales radio 3.");
+            //ALTA VARIANTA DE REZOLVARE
+//            File file = driver.getScreenshotAs(OutputType.FILE);
+//            File destinationFile = new File("C:\\Users\\Deniz\\IdeaProjects\\poza2.png");
+//            FileUtils.copyFile(file, destinationFile);
+        } finally {
+            File file = driver.getScreenshotAs(OutputType.FILE);
+            File destinationFile = new File("C:\\Users\\Deniz\\IdeaProjects\\poza2.png");
```

Comment:
Use FileUtils.takesScreenshot method instead of these lines to avoid code duplication. Thanks!

Reply:
done